website/pages/docs/setup.mdx

```diff
@@ -242,183 +200,115 @@ yarn prepare
-yarn typia generate \
-  --input src/templates \
-  --output src/generated \
-  --project tsconfig.json
+yarn typia setup --manager yarn
 ```
   </Tab>
 </Tabs>
 
-For frontend projects.
-
-If you are using a non-standard TypeScript compiler such as the following, you will need to fall back to [generation](#generation) mode
-
-  - Non-standard TypeScript compilers:
-    - [SWC](https://swc.rs/) in Next.JS
-    - [esbuild](https://esbuild.github.io/) in Vite
-    - [Babel](https://babeljs.io/) in Create-React-App
-
-Instead you should utilise the generation mode. 
-
-Install `typia` through `npm install` command, and run `typia generate` command. Then, generator of `typia` reads your TypeScript codes of `--input`, and writes transformed TypeScript files into the `--output` directory, like below.
+At first, install both `unplugin-typia` and `typia`, with `npx typia setup` command.
 
-For clarification, the input directory should contain one or more TypeScript files which define how you want to verify your associated type assertions. Commonly you will import your TypeScript type, then export a function which validates that type. See below.
+After that, follow the next section steps to integrate `unplugin-typia` into your bundlers.
 
-If you want to specify other TypeScript project file instead of `tsconfig.json`, you can use `--project` option.
+For reference, there are a couple of ways to integrate `unplugin-typia` into your bundlers. For the full integration guide, please refer to the [`unplugin-typia` documentation](https://jsr.io/@ryoppippi/unplugin-typia/doc). Also, you can see the examples projects in [`unplugin-typia` repository](https://github.com/ryoppippi/unplugin-typia).
 
-<Tabs items={['TypeScript Source Code', 'Generated TypeScript File']}>
+<Tabs items={['Vite', 'Next.js', 'esbuild']}>
   <Tab>
-```typescript copy filename="examples/src/templates/check.ts" showLineNumbers {5}
-import typia from "typia";
-
-import { IMember } from "../structures/IMember";
-
-export const check = typia.createIs<IMember>();
+```typescript filename="vite.config.ts" copy showLineNumbers
+import UnpluginTypia from 'unplugin-typia/vite'
+ 
+export default defineConfig({
+  plugins: [UnpluginTypia({ /* options */ })],
+})
 ```
   </Tab>
   <Tab>
-```typescript copy filename="examples/src/generated/check.ts" showLineNumbers {3-13}
-import typia from "typia";
-import { IMember } from "../structures/IMember";
-export const check = (input: any): input is IMember => {
-  const $is_uuid = (typia.createIs as any).is_uuid;
-  const $is_email = (typia.createIs as any).is_email;
-  return (
-    "object" === typeof input &&
-    null !== input &&
-    "string" === typeof input.id &&
-    $is_uuid(input.id) &&
-    "string" === typeof input.email &&
-    $is_email(input.email) &&
-    "number" === typeof input.age &&
-    19 <= input.age &&
-    100 >= input.age
-  );
+```javascript filename="next.config.mjs" copy showLineNumbers
```

Comment:
I wrote more detailed version of config for Next.js. so please replace it.
https://jsr.io/@ryoppippi/unplugin-typia/doc/next/~/default